SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -123,10 +124,6 @@ void USpatialReceiver::OnAddEntity(const Worker_AddEntityOp& Op)
 {
 	SCOPE_CYCLE_COUNTER(STAT_ReceiverAddEntity);
 	UE_LOG(LogSpatialReceiver, Verbose, TEXT("AddEntity: %lld"), Op.entity_id);
-
```

Comment:
Er, is this really a no-op now?

Reply:
what else should it do? Previously it was only used to indicate an actor should be spawned within this critical section. That has now moved to the metadata component, and previously the metadata was a noop, so should be fine?

Reply:
Surprised there was no book keeping required at this level, but makes sense!
Can you do a driveby and remove the performance scope and marker? (STAT_ReceiverAddEntity)

Reply:
removed this scope cycle counter line and the declare cycle stat at the top of the file